src/schema.rs

```diff
@@ -64,7 +64,7 @@ impl TryCmd {
                 sequence
             } else if ext == std::ffi::OsStr::new("trycmd") || ext == std::ffi::OsStr::new("md") {
                 let raw = crate::Data::read_from(path, Some(snapbox::DataFormat::Text))?
-                    .map_text(snapbox::utils::normalize_lines)
+                    .normalize(snapbox::NormalizeNewlines)
```

Comment:
This is an abuse of `Data` that we were supposed to move off of before this change

Reply:
In general, I'd recommend doing a scrub of changes to this file to make sure we are only using Data when we should and only using `try_string` when we should